### ISSUE TITLE
Fix metalint issues

### ISSUE THREAD

**Issue by mperham** [mperham/inspeqtor-pro#1]
`make lint` shows ~100 warnings.  Investigate and fix as necessary.

**Comment by sorentwo** [comment]
The majority of warnings look to come from generated files, namely parser/token packages. From what I can see `gometalinter` only accepts a single or recursive path, it doesn't accept patterns for files to ignore. Added to that, running the linters on a subdirectory yields rampant type errors because none of the associated packages can be found. This seems like an uphill battle.

**Comment by mperham** [comment]
Yeah, that's the investigate part of the task.  Too bad it's mostly noise.  We can give it a subset of packages excluding the parser stuff.
